apps/electron/package.json

```diff
@@ -44,6 +45,7 @@
     "electron-window-state": "^5.0.3",
     "esbuild": "^0.17.18",
     "fs-extra": "^11.1.1",
+    "js-yaml": "^4.1.0",
```

Comment:
I think you did not use this library in generate-yml. Maybe remove it or generate JSON and convert it to yml using this lib instead?

Reply:
will use fs.writeFileSync to write a yml string